lib/components/Input/base/index.tsx

```diff
@@ -164,6 +173,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
 
                     {triggerActionIcon && <>{triggerActionIcon}</>}
                 </div>
+                {hasDropdown && isDropdownOpen && !disabled && children}
```

Comment:
I think children from dropdown should have some kind of container with positioning (in order not to shift neighboring elements)

Reply:
+ According to design, dropdown should have title (not only placeholder) and arrow icon, which will indicate if dropdown is expanded or collapsed. 
Taking in account all this maybe it's better to create a separate component, based on ./base/index.tsx?
![Screenshot 2024-05-10 at 11 25 18 AM](https://github.com/deriv-com/quill-ui/assets/121025168/022700df-a5a9-4fa3-bd0e-23054f727dd1)